internal/internal_workflow_client.go

```diff
@@ -308,7 +313,7 @@ func (wc *workflowClient) GetWorkflow(ctx context.Context, workflowID string, ru
 }
 
 // SignalWorkflow signals a workflow in execution.
-func (wc *workflowClient) SignalWorkflow(ctx context.Context, workflowID string, runID string, signalName string, arg interface{}) error {
+func (wc *workflowClient) SignalWorkflow(ctx context.Context, workflowID string, runID string, signalName string, arg interface{}, opts ...SignalWorkflowOption) error {
```

Comment:
if we expose options, we should try and stick to the standard options pattern in Go 
see example here - https://gist.github.com/travisjeffery/8265ca411735f638db80e2e34bdbd3ae

Reply:
@mfateev I think this is reasonable suggesion. We also want to hear if you have any more ideas.

Reply:
We started from this options standard pattern. The user feedback was very negative. Everyone asked for a structure.